ci/scripts/clone-build_ci.sh

```diff
@@ -73,37 +68,54 @@ if [[ "${pr_state}" != "OPEN" ]]; then
 fi  
  
 # checkout pull request
-"${GH}" pr checkout "${PR}" --repo "${repo_url}"
+"${GH}" pr checkout "${PR}" --repo "${REPO_URL}"
 
 # get commit hash
 commit=$(git log --pretty=format:'%h' -n 1)
 echo "${commit}" > "../commit"
 
 # run build script
-cd sorc
+cd sorc || exit
```

Comment:
The `exit` should be `non-zero` in case of a failure.
```suggestion
cd sorc || exit 1
```

Reply:
Good one! Pun not intended. ;)